disk-mapper/recoverproto/recover.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.34.1
-// 	protoc        v4.22.1
+// 	protoc        v3.21.7
```

Comment:
Is the protoc version downgrade intentional?

Reply:
Yes. We had a custom fork of rules_proto previously, because we were on a old version that had a patch to support statically linked protoc.

We now adopted the new and improved [rules_proto 6](https://github.com/bazelbuild/rules_proto/releases/tag/6.0.0).